diff --git a/node_modules/react-native-css-interop/.cache/android.js b/node_modules/react-native-css-interop/.cache/android.js
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/react-native-css-interop/.cache/ios.js b/node_modules/react-native-css-interop/.cache/ios.js
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/react-native-css-interop/.cache/macos.js b/node_modules/react-native-css-interop/.cache/macos.js
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/react-native-css-interop/.cache/native.js b/node_modules/react-native-css-interop/.cache/native.js
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/react-native-css-interop/.cache/windows.js b/node_modules/react-native-css-interop/.cache/windows.js
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/react-native-css-interop/dist/runtime/native/render-component.js b/node_modules/react-native-css-interop/dist/runtime/native/render-component.js
index eba25ec..56c1924 100644
--- a/node_modules/react-native-css-interop/dist/runtime/native/render-component.js
+++ b/node_modules/react-native-css-interop/dist/runtime/native/render-component.js
@@ -121,22 +121,32 @@ function printUpgradeWarning(warning, originalProps) {
     console.log(`CssInterop upgrade warning.\n\n${warning}.\n\nThis warning was caused by a component with the props:\n${stringify(originalProps)}\n\nIf adding or removing sibling components caused this warning you should add a unique "key" prop to your components. https://react.dev/learn/rendering-lists#keeping-list-items-in-order-with-key\n`);
 }
 function stringify(object) {
-    const seen = new WeakSet();
-    return JSON.stringify(object, function replace(_, value) {
-        if (!(value !== null && typeof value === "object")) {
-            return value;
-        }
-        if (seen.has(value)) {
-            return "[Circular]";
-        }
-        seen.add(value);
-        const newValue = Array.isArray(value) ? [] : {};
-        for (const entry of Object.entries(value)) {
-            newValue[entry[0]] = replace(entry[0], entry[1]);
-        }
-        seen.delete(value);
-        return newValue;
-    }, 2);
+    try {
+        const seen = new WeakSet();
+        return JSON.stringify(object, function replace(_, value) {
+            if (!(value !== null && typeof value === "object")) {
+                return value;
+            }
+            if (seen.has(value)) {
+                return "[Circular]";
+            }
+            seen.add(value);
+            const newValue = Array.isArray(value) ? [] : {};
+            try {
+                for (const entry of Object.entries(value)) {
+                    newValue[entry[0]] = replace(entry[0], entry[1]);
+                }
+            }
+            catch {
+                return "[Non-serializable object]";
+            }
+            seen.delete(value);
+            return newValue;
+        }, 2);
+    }
+    catch {
+        return "[Unable to stringify props]";
+    }
 }
 function getDebugReplacer() {
     const seen = new WeakSet();
@@ -147,10 +157,10 @@ function getDebugReplacer() {
             }
             seen.add(value);
             if ("_isReanimatedSharedValue" in value && "value" in value) {
-                return `${value.value} (animated value)`;
+                return "[SharedValue]";
             }
             if ("get" in value && typeof value.get === "function") {
-                return value.get();
+                return "[AnimatedNode]";
             }
         }
         else if (typeof value === "function") {
