diff --git a/node_modules/react-native-enriched-html/lib/module/web/formats/EnrichedLink.js b/node_modules/react-native-enriched-html/lib/module/web/formats/EnrichedLink.js
index d552741..0646430 100644
--- a/node_modules/react-native-enriched-html/lib/module/web/formats/EnrichedLink.js
+++ b/node_modules/react-native-enriched-html/lib/module/web/formats/EnrichedLink.js
@@ -51,8 +51,9 @@ export const EnrichedLink = Link.extend({
     return {
       ...parent,
       openOnClick: false,
-      autolink: false,
-      linkOnPaste: false,
+      autolink: true,
+      linkOnPaste: true,
+      defaultProtocol: 'https',
       HTMLAttributes: {
         ...parent.HTMLAttributes,
         target: null,
